core/src/test/resources/data/responses/invalids/invalid_subjectconfirmation_nb.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
```

Comment:
Same as `core/src/test/resources/data/responses/invalids/invalid_subjectconfirmation_nb.xml.base64`, just not base64 encoded and added `NotOnOrAfter` to pass the new validation, and fail on the NotBefore check.